plugins/out_s3/s3.c

```diff
@@ -405,6 +405,9 @@ static int cb_s3_init(struct flb_output_instance *ins,
             flb_plg_error(ctx->ins, "'s3_key_format' must start with a '/'");
             return -1;
         }
+        if (strstr((char *) tmp, "$UUID")) {
+            ctx->support_uuid = FLB_TRUE;
```

Comment:
nit: I don't think `support_uuid` is an ideal variable name. This is a boolean, and the best naming schema for booleans is to do add the word "has" or "is". So `has_uuid` or `key_fmt_has_uuid`... something along those lines